### PR TITLE
onboarding task #1 solution (issue #90)

### PR DESCRIPTION
**Pull request by hakanayata**

### CODE REVIEW

File: packages/ui/src/examples/E0_ResponsiveCard.tsx [hunk 1]
```diff
@@ -4,30 +4,26 @@ export const ResponsiveCard = () => {
   const randomPic = 'https://picsum.photos/600/400'
 
   return (
-    <Flex>
-      <Stack>
-        <Image src={randomPic} alt="random pic" />
-      </Stack>
+      <Flex shadow="lg" direction={{base: "column", md: "row"}}>
```
Comment (7alip) [inline packages/ui/src/examples/E0_ResponsiveCard.tsx:7]: The flex direction changes at the md breakpoint, but the image width changes at the sm breakpoint. This explains the layout issue:

Screenshot 2023-09-17 at 23 35 21

**Review by 7alip — COMMENTED** [review]
I see that your code is not formatted.

We have the following settings in `.vscode/settings.json`. Your code should be formatted each time you save it. Maybe you don't have VSCode Eslint extension.

```json
{
  "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   }
  }

File: packages/ui/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "extends": "@wsvvrijheid/config/tsconfig.nextjs.json",
   "include": ["next-env.d.ts", "@types/**/*.ts", "**/*.ts", "**/*.tsx"],
-  "exclude": ["dist", "build", "node_modules"]
+  "exclude": ["dist", "build", "node_modules"],
+  "compilerOptions": {
+    "jsx": "react-jsx"
```
[the remark this answers is not in the document]
Comment (hakanayata) [reply packages/ui/tsconfig.json:6]: There was a TypeScript compiler error. However, I've just commented the lines 5-7 and it seems to be working fine now. Maybe my initial `npm install` caused this as well.

**Review by 7alip — APPROVED** [review]
It is not a problem for this task, but for your information, if `Ctrl + S` does not work for you, you can format your code with `yarn format`.

Thanks, you can close the PR.

**Comment by hakanayata** [comment]
> It is not a problem for this task, but for your information, if `Ctrl + S` does not work for you, you can format your code with `yarn format`.
> 
> Thanks, you can close the PR.

Thank you for pointing this out. I had Prettier and Eslint installed but didn't have the "default formatter" configured. 
image

I opted for  "TypeScript and JavaScript Language Features" as my formatter and I also added the lines below from your message to my settings.json. 
```json
  "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   }
```
"Format on save" seems to be working now. Thank you very much.

**Pull request closed without merging**